core/src/hooks/useParticipants.ts

```diff
@@ -7,37 +7,49 @@ import {
   useState,
 } from "react";
 import GlobalClientProviderContext from "../context/LMGlobalClientProviderContext";
-
-import { LMChatChatroomContext } from "../context/LMChatChatroomContext";
+import { LMChatroomContext } from "../context/LMChatChatroomContext";
 import { ZeroArgVoidReturns } from "./useInput";
-import {
-  Participant,
-  ViewParticipantsResponse,
-} from "../types/api-responses/viewParticipants";
-import { useNavigate, useParams } from "react-router-dom";
-// import { CHANNEL_PATH } from "../shared/constants/lm.routes.constant";
+import { ViewParticipantsResponse } from "../types/api-responses/viewParticipants";
+
+import Member from "../types/models/member";
+import { CustomisationContextProvider } from "../context/LMChatCustomisationContext";
 
 /**
  * Custom hook that provides functionality related to participants/members of a chatroom.
  * @returns {UseParticipantsReturns} An object containing the participants list, a flag indicating whether there are more participants to load, a function to fetch the members/participants of a chatroom, and a function to navigate back to the chatroom.
  */
 export function useParticipants(): UseParticipantsReturns {
-  const { lmChatclient, routes } = useContext(GlobalClientProviderContext);
-  const { chatroom } = useContext(LMChatChatroomContext);
-  const [participantsList, setParticipantList] = useState<Participant[]>([]);
+  const { lmChatClient } = useContext(GlobalClientProviderContext);
+  const { participantsCustomActions = {} } = useContext(
+    CustomisationContextProvider,
+  );
+  const {
+    getMembersCustomCallback,
+    navigateBackToChatroomCustomCallback,
+    setSearchKeywordCustomCallback,
+  } = participantsCustomActions;
+  const { chatroomDetails } = useContext(LMChatroomContext);
+  const [participantsList, setParticipantList] = useState<Member[]>([]);
   const participantListPageCount = useRef<number>(1);
   const [searchKeyword, setSearchKeyword] = useState<string>("");
   const totalParticipantsCount = useRef<number>(0);
   const [loadMoreParticipants, setLoadMoreParticipants] =
     useState<boolean>(true);
-  const { id: chatroomId } = useParams();
-  const navigate = useNavigate();
+  const {
+    chatroomDetails: {
+      // TODO
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      chatroom: { id: chatroomId },
+    },
+  } = useContext(LMChatroomContext);
+
   /**
    * Navigates back to the chatroom.
    */
   const navigateBackToChatroom = useCallback(() => {
-    navigate(`/${routes?.getChannelPath()}/${chatroomId}`);
-  }, [chatroomId, navigate, routes]);
+    window.history.back();
+    // TODO
```

Comment:
why todo?

Reply:
Removed. It was supposed to be removed earlier